easybuild/easyconfigs/p/PyTorch-bundle/PyTorch-bundle-1.12.1-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,59 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-bundle'
```

Comment:
As Micket didn't have a strong opinion (see below) I'd say that you can probably decide @boegel. To add some nuance I'd like to note that it is a PythonBundle of PyTorch domain libraries under https://github.com/pytorch, though excluding PyTorch it self, and as it is a bundle these libraries are added in an extension list. So, I think PyTorch-extensions and PyTorch-bundle can both make sense, or even PyTorch-extensions-bundle if you'd like to be even more specific.

I looked through the names of other bundles and these where the only other relevant ones I found (i.e. they didn't just have names corresponding to one of the dependencies or extensions):
 - ESL-Bundle
 - R-bundle-Bioconductor
 - SciPy-bundle

As this is the only thing remaining I'd appreciate if it could be resolved soon.

Reply:
My vote goes for Python-bundle